internal/pkg/service/buffer/api/service/export.go

```diff
@@ -0,0 +1,148 @@
+package service
```

Comment:
`service` som rozdelil do viacerych suborov `receiver.go`, `export.go`, ...

A vdaka predchadzajucim zmenam je to teraz:
- prehladnejsie a kratsie
- je podporovany rollback
- jednotlive operacie su delegovane na `file/table/token.Manager` a tam su aj unit-testovane